src/backend/distributed/utils/citus_outfuncs.c

```diff
@@ -482,6 +482,7 @@ OutTask(OUTFUNC_ARGS)
 	WRITE_UINT64_FIELD(anchorShardId);
 	WRITE_NODE_FIELD(taskPlacementList);
 	WRITE_NODE_FIELD(dependentTaskList);
+	WRITE_NODE_FIELD(queryStringList);
```

Comment:
Minor: These are generally in the order of the definitions (i.e. after queryStringLazy).

We no longer have citus_readfuncs.c, but if we ever find a need to bring it back it's important to be consistent between the out function and the read function, and following the order of the struct is an easy way to remain consistent.